README.md

```diff
@@ -18,6 +81,43 @@ Non exaustive list:
 - pkg-config
 - C Compiller
 
-## contribution
+
+## Contribution
 
 Open Pull Requests and Issues are welcome.
+
+If you want to add binary in bin/ please try to make it easy to compile.
+That mean, don't add dependencies that are not already require by a CLI.
```

Comment:
This means avoiding additional dependencies that are not already required by the CLI.